src/main/java/frc/robot/subsystems/Intake/IntakeState.java

```diff
@@ -0,0 +1,17 @@
+package frc.robot.subsystems.Intake;
+
+import frc.robot.Direction;
+
+public enum IntakeState {
+    IDLE(false, Direction.OFF),
+    INTAKE(true, Direction.RUNNING),
+    REVERSE(true, Direction.REVERSED);
+
+    public boolean intakeExtended;
```

Comment:
you may not be able to get away with a boolean value for this, just check with the design peeps that the intake will be in the normal "extended" position while carrying the ball